### PR TITLE
proton-ge: 8.6 -> 8.9

### PR DESCRIPTION
**Pull request by NotAShelf**
contains hotfixes to 8.8

### CODE REVIEW

File: pkgs/proton-ge/default.nix [hunk 1]
```diff
@@ -5,11 +5,12 @@
 }:
 stdenv.mkDerivation (finalAttrs: {
   pname = "proton-ge-custom";
-  version = "GE-Proton8-6";
+
```
Comment (fufexan) [inline pkgs/proton-ge/default.nix:8]: Why is this here?

Comment (NotAShelf) [reply pkgs/proton-ge/default.nix:8]: git works in mysterious ways

**Comment by fufexan** [comment]
There we go, thanks!

**Comment by NotAShelf** [comment]
no
